configuration/wireless/wireless-cli.md

```diff
@@ -45,11 +45,11 @@ The password can be configured either as the ASCII representation, in quotes as
   ```
 Note that the plain text version of the code is present, but commented out. You should delete this line from the final `wpa_supplicant` file for extra security.
 
-The `wpa_passphrase` tool requires a password with between 8 and 63 characters. For more complex passphrases you can extract the content of a text file and use it as input for `wpa_passphrase`, if the password is stored as plain text inside a file somewhere, by calling `wpa_passphrase "testing" < file_where_password_is_stored`. For extra security, you should delete the `file_where_password_is_stored` afterwards, so there is no plain text copy of the original password on the system.
+The `wpa_passphrase` tool requires a password with between 8 and 63 characters. To use a more complex password, you can extract the content of a text file and use it as input for `wpa_passphrase`. Store the password in a text file and input it to `wpa_passphrase` by calling `wpa_passphrase "testing" < file_where_password_is_stored`. For extra security, you should delete the `file_where_password_is_stored` afterwards, so there is no plain text copy of the original password on the system.
 
-To use the `wpa_passphrase`–encrypted PSK, you can either copy and paste the encrypted PSK into the `wpa_supplicant.conf` file, or redirect the tools output to the configuration file in one of two ways:
+To use the `wpa_passphrase`–encrypted PSK, you can either copy and paste the encrypted PSK into the `wpa_supplicant.conf` file, or redirect the tool's output to the configuration file in one of two ways:
 - Either change to `root` by executing `sudo su`, then call `wpa_passphrase "testing" >> /etc/wpa_supplicant/wpa_supplicant.conf` and enter the testing password when asked
-- Or use `wpa_passphrase "testing" | sudo tee -a /etc/wpa_supplicant/wpa_supplicant.conf > /dev/null` and enter the testing password when asked; the redirection to `/dev/null` prevents `tee` from **also** outputting to the screen (standard output)
+- Or use `wpa_passphrase "testing" | sudo tee -a /etc/wpa_supplicant/wpa_supplicant.conf > /dev/null` and enter the testing password when asked; the redirection to `/dev/null` prevents `tee` from **also** outputting to the screen (standard output).
 
 If you want to use one of these two options, **make sure you use `>>`, or use `-a` with `tee`** — either will **append** text to an existing file. Using a single chevron `>`, or omitting `-a` when using `tee`, will erase all contents and **then** append the output to the specified file.
```

Comment:
For the newbie reading these instructions, would it be beneficial to only recommend one way of doing things rather than giving them multiple options? @JamesH65 ?

Reply:
Hmm, dunno. The phrasing is clear - one or the other, so I am not really bothered either way. We are here to teach!

Reply:
My preference is one way of doing things in the docs. It's a bit confusing at the moment I feel. It seems like there's a whole load of stuff there that's not actually necessary, but I didn't want to get bogged down in having a big discussion about it. If you would like me to remove some things I'm happy to do that.

Reply:
> It's a bit confusing at the moment I feel. It seems like there's a whole load of stuff there that's not actually necessary

That's the trouble with community-written documentation - everyone thinks that their way of doing things is best, so you end up with many PRs adding lots of different ways of doing things :wink: 

If James is happy with the current content, then I'm not going to suggest changing it.